openapi/openapi.yaml

```diff
@@ -0,0 +1,228 @@
+openapi: 3.0.3
+info:
+  title: User API for Speakeasy template service
+  description: The Rest Template API is an API used for instrucitonal purposes.
+  version: 0.1.0
+  contact:
+    name: Speakeasy DevRel
+    email: nolan@speakeasyapi.dev
+servers:
+  - url: http://localhost:8080
+paths:
+  /v1/user:
+    post:
+      operationId: createUserv1
```

Comment:
I would avoid adding `v1` type suffixs and only start using version in operation names when there exists more than one version for something then start at v2